sim/common/sod/item_effects/phase_3.go

```diff
@@ -89,6 +92,110 @@ func init() {
 		})
 	})
 
+	core.NewItemEffect(DarkmoonCardDecay, func(agent core.Agent) {
+		character := agent.GetCharacter()
+
+		makeDecayAura := func(target *core.Unit, playerLevel int32) *core.Aura {
+			return target.GetOrRegisterAura(core.Aura{
+				Label:    "Decay",
+				ActionID: core.ActionID{SpellID: 446393},
+				// Placeholder duration
+				Duration:  core.NeverExpires,
+				MaxStacks: 5,
+			})
+		}
+
+		decayAuras := character.NewEnemyAuraArray(makeDecayAura)
+
+		decayStackedSpell := character.RegisterSpell(core.SpellConfig{
+			ActionID:    core.ActionID{SpellID: 446810},
+			SpellSchool: core.SpellSchoolShadow,
+			DefenseType: core.DefenseTypeMagic,
+			ProcMask:    core.ProcMaskEmpty,
+
+			DamageMultiplier: 1,
+			ThreatMultiplier: 1,
+		})
+
+		decayProcSpell := character.RegisterSpell(core.SpellConfig{
+			ActionID:    core.ActionID{SpellID: 446393},
+			SpellSchool: core.SpellSchoolShadow,
+			DefenseType: core.DefenseTypeMagic,
+			ProcMask:    core.ProcMaskEmpty,
+
+			DamageMultiplier: 1,
+			ThreatMultiplier: 1,
+
+			ApplyEffects: func(sim *core.Simulation, target *core.Unit, spell *core.Spell) {
+				targetAura := decayAuras[target.Index]
+				// Placeholder damage and coefficient values, update when P3 releases
+				result := spell.CalcAndDealDamage(sim, target, 30+0.05*spell.SpellDamage(), spell.OutcomeMagicHitAndCrit)
+				if result.Landed() {
+					spell.CalcAndDealHealing(sim, &character.Unit, result.Damage, spell.OutcomeHealing)
+					targetAura.Activate(sim)
+					targetAura.AddStack(sim)
+				}
+				if targetAura.GetStacks() == 5 {
+					// Placeholder coefficient
+					decayStackedSpell.CalcAndDealDamage(sim, target, sim.Roll(120, 180)+0.05*spell.SpellDamage(), spell.OutcomeMagicHitAndCrit)
```

Comment:
It looks like Darkmoon Card: Maelstrom doesn't have a SP coefficient listed https://www.wowhead.com/classic/spell=23687/lightning-strike. Do you know if it actually has one? I wonder if we should default to no coefficient for now